pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -249,6 +252,10 @@ self: super: {
   ncm2-ultisnips = super.ncm2-ultisnips.overrideAttrs(old: {
     dependencies = with super; [ ultisnips ];
   });
+  
+  notational-fzf-vim = super.notational-fzf-vim.overrideAttrs(old: {
+    dependencies = with super; [ ripgrep fzf fzf-vim ];
```

Comment:
I would say `with self` instead as we want to use the last defined packages.

Reply:
If I recall correctly, `dependencies` is only supposed to be used for dependencies on other vim plugins. You probably want patch the plugin instead (hard-coding the ripgrep path or appending to `PATH`).